AnnoDesigner/Constants.cs

```diff
@@ -12,7 +12,7 @@ public static class Constants
         /// Version number of the application.
         /// Will be increased with each release.
         /// </summary>
-        public const double Version = 8.81;
+        public static readonly Version Version = new Version(8, 81);
```

Comment:
Worth adding a comment here that mentions what you've said in the PR about intentionally keeping `major.minor` for a little while.